src/main/java/Nateh.java

```diff
@@ -0,0 +1,228 @@
+import classes.Deadlines;
+import classes.Event;
+import classes.Task;
+import classes.Todo;
+import exceptions.IllegalCommandException;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.IOException;
+import java.util.Scanner;
+
+public class Nateh {
+    
+    public static void printList(ArrayList<Task> list) {
+        System.out.print(Skeleton.LINE_BREAK);
+        for (int i = 0; i < list.size(); i++) {
+            System.out.print(i + 1 + ". ");
+            list.get(i).print();
+        }
+        System.out.print(Skeleton.LINE_BREAK);
+    }
+
+    public static void handleMark(String[] splitInput, ArrayList<Task> list) {
```

Comment:
Some duplicated code blocks here, perhaps can try and refactor them to improve code readability